action.yml

```diff
@@ -48,6 +48,11 @@ runs:
         GH_TOKEN: ${{ inputs.token }}
       shell: bash
       run: |
+        if [ "$GH_TOKEN" = "" ]; then
+          echo "Error! For destination=default you have to pass token input also."
```

Comment:
Change message to: "Please provide the `secrets.GITHUB_TOKEN` as input for GitHub Pages deployment."

Reply:
Done